### PR TITLE
Don't emit any column infos for CodeView by default, like clang

### PR DESCRIPTION
**Pull request by kinke**
Fixing #3102; see https://reviews.llvm.org/D23720.

### CODE REVIEW

File: gen/dibuilder.cpp [hunk 1]
```diff
@@ -97,6 +97,12 @@ bool DIBuilder::mustEmitLocationsDebugInfo() {
   return (global.params.symdebug > 0) || global.params.outputSourceLocations;
 }
 
+unsigned DIBuilder::getColumn(const Loc &loc) {
+  // like clang, don't emit any column info for CodeView
+  // (https://reviews.llvm.org/D23720)
+  return (loc.linnum && !isTargetMSVC) ? loc.charnum : 0;
```
Comment (PetarKirov) [inline gen/dibuilder.cpp]: Perhaps make this configurable, like clang's `-gcolumn-info`?

Comment (kinke) [reply gen/dibuilder.cpp]: I can't think of any use for it, so I decided against a new option.

Comment (PetarKirov) [reply gen/dibuilder.cpp]: I don't have a use case, so it's not important to me. I'm just wary of unconditionally removing information, without giving the user a choice.
-> Feel free to keep it like this, and add it later if someone else actually needs it.

Comment (kinke) [reply gen/dibuilder.cpp]: The clang PR mentions that it's still useful for sanitizers, so the option is available now.

File: tests/debuginfo/msvc_no_colinfo.d [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// REQUIRES: Windows
```
Comment (kinke) [inline tests/debuginfo/msvc_no_colinfo.d]: Only restricted to Windows hosts to test the default setting.

File: gen/dibuilder.cpp (hunk 1 above)
Comment (PetarKirov) [reply gen/dibuilder.cpp]: > The clang PR mentions that it's still useful for sanitizers, so the option is available now.

Yes, that's why I opened the question regarding the option, sorry if it was unclear.

**Pull request merged**